third_party/blink/renderer/bindings/core/v8/local_window_proxy.h

```diff
@@ -114,8 +114,6 @@ class LocalWindowProxy final : public WindowProxy {
     return To<LocalFrame>(WindowProxy::GetFrame());
   }
 
-  mojo::Remote<auth_token::mojom::RecordReplayAuthTokenStore> auth_token_store_;
-  
```

Comment:
and this file's change is the other that's not renaming.  The window proxy creates a `RecordReplayEventListener`, which routes events through its `LocalFrame`, which then delegates to its `LocalFrameMojoHandler` to deal with all the mojo stuff.